example/README.md

```diff
@@ -1,4 +1,19 @@
-# Example Application
+# @pixeloven/example
 
-### Todos
-- Need to have @pixeloven packages point to local or at-least master committed versions instead of npm
\ No newline at end of file
+> Pixel Oven core.
+
+See our website [pixeloven-core](https://github.com/pixeloven/pixeloven) for more information or the [issues](https://github.com/pixeloven/pixeloven) associated with this package.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save @pixeloven/example
```

Comment:
`--save` flag isn't necessary, the default for `npm install` has been to save since NPM 5